papers/Raku_Steering_Council_Code.md

```diff
@@ -54,63 +46,92 @@ The Steering Council is a 7-person committee.
+Code of Conduct committee.
+
+The Steering Council may only use the powers granted to it by taking a vote on a formal proposal. 
+Every council member must either vote or explicitly abstain. Members with conflicts of interest on
 a particular vote must abstain. Passing requires support from a majority
 of non-abstaining council members.
 
-Whenever possible, the council's deliberations and votes shall be held
-in public.
+Minutes shall be made available to the Core Development Team for each meeting of the Steering Council, 
+which shall contain a description of a topic before the
+Steering Council, the possible alternative solutions considered by the Steering Council, the formal proposal
+leading to a decision, and the votes of Council members.
+
+Minutes about topics that contain confidential information, such as personal information or sensitive intrapersonal topics, 
+may be withheld from publication. The over-riding motivation for confidential minutes is to protect personal information, as
+is required by the legislation of many countries. Any topic involving extensive confidential information shall be
+delegated to a subcommittee to investigate and to make recommendations for action by the Steering Council.
 
+Minutes are published by placing them on a link on the Raku website and a message about the link is sent to the Raku users
+email list.
 
 ### Electing the council
 
-A council election consists of two phases:
+A council election has phases:
 
 * Phase 1: Candidates advertise their interest in serving. Candidates
 must be nominated by a core team member. Self-nominations are
-allowed.
-
-* Phase 2: Each core team member can vote for zero to five of the
-candidates. Voting is performed anonymously. Candidates are ranked
+allowed. Candidates should be a member of the Raku Users' Community. 
+Candidate names shall be made distinct by the Steering Council prior
+to publication of the Candidate list, eg., by adding a work place, or residence
+town to the end of two identical names. Candidates must declare their
+place of employment and clearly indicate whether they are working for the 
+same company as another candidate.
+
+* Phase 2: Each core team member can vote for zero to five distinct 
+candidates. Voting is performed anonymously. Duplicate names on the same ballot
+shall be counted as a single vote for that candidate. Names other than for the
+recognised candidates shall not be counted. If the name contained in a ballot is ambiguous
+it shall not be counted.
+
+* Counting: Candidates are ranked
 by the total number of votes they receive. If a tie occurs, it may
-be resolved by mutual agreement among the candidates, or else the
-winner will be chosen at random.
+be resolved by mutual agreement among the candidates, or by the selection of the
+candidate from a minority, such as a woman if the majority of other council members
+are male, or from a country not otherwise represented on the Council, or else the
```

Comment:
Is this really the best way to handle it?

Reply:
We see in the world right NOW that institutional racism and bias can hurt and deform mature societies. There remains a glass ceiling for women. Why is it that with 50% of the population, and certainly some extremely clever women, our group contains only one female? In order to prevent even the possibility of discrimination and bias, there needs to be an extra effort to overcome it. That is the primary reason I have suggested this. Remember is only for the relatively rare cases when there is a tie vote for the last place(s). But even if there were no prejudice in our society, including minorities increases diversity and that has shown repeatedly to be beneficial to the whole group.

Reply:
> our group contains only one female

Only one?  Are you sure?

Reply:
Having an open-ended way to select the tie breaker makes the process more complicated - how and when do you select the tie breaker? A simpler method is to require an odd # of members.

Reply:
First of all, I'm not so much concerned about the gender thing, but more about the useless note about countries (and I guess implicitly about other unrelated differences in people that somehow should affect the results).

> In order to prevent even the possibility of discrimination and bias

… you pretty much introduce discrimination and bias. I understand that you're trying to do it with good intent to help fix the situation, but that's not the nicest approach.

> Remember is only for the relatively rare cases when there is a tie vote

Then just cover this relatively rare edge case with a mechanism that is already well described – use an additional voting period for the tie.

Remember, the council itself is unlikely to have a good representation for all minorities (especially wrt countries, I don't know how you even put that in), so biasing towards minorities for ties *sounds* like you're doing something but in reality its effect is minuscule. Of course, we will all hope that the selected council will do their best to make decisions that take minorities into account, but the council itself can't possibly be a good representation of all minorities – it's just 7 people.

Reply:
Also, I applaud your attempt at designing a system that by itself protects interests of minorities through its mechanisms, but you're off with a bad start. The “majority wins” idea for the 7 member council is inherently unable to put up a fight against discrimination.

The problem-solving repo solves that by putting infinite weight on negative votes, meaning that a change can be prevented by any person if it hurts their minority, or by a person who decides to represent a minority. Yes, it shifts the very hard issue of discrimination into a potential problem with trolling, but the latter is much easier to deal with.

Reply:
Actually, the benefit of a minority - however defined - comes not in voting, but in the voices during discussions, and a wider variety of ideas and solutions. People get fixated on the voting, but most good decision-making is by consensus, and having more variety of experience generates more opportunities to build consensus.

The rationales so far expressed here not to include some form of extra inclusion (and again I say it is a marginal extra) are the very same rationales that white men have used to exclude women and minorities from the decision-making process until now.

Since we are at a very interesting point in time in the history of the world (I am being utterly sincere in this assessment), the mere fact that the Raku community would take an extra step towards inclusion would also have significant knock-on effects.

Reply:
The reason why I wouldn't like this change be included is that we're messing up the material world with Internet. How do we know if the person we're about to choose belongs to a minority? For example, all I know about zoffix is the nick; and that the nick pretends to be from Toronto, Canada, but even this is not certain. I neither know the gender, nor anything else. Perhaps some of community members know more. But if some doesn't want they personal details to be published – will the procedure exclude them as likely candidates?

From this point of view I'd rather prefer dice-thrown approach. It's way more fair.

Reply:
> Actually, the benefit of a minority - however defined - comes not in voting, but in the voices during discussions, and a wider variety of ideas and solutions. People get fixated on the voting, but most good decision-making is by consensus

I know. It's one of the reasons why the problem-solving repo doesn't even have traditional voting, among other things. It forces people improve their proposed solutions, with no way of ignoring the voices by falling back to the shortcut of voting. This is also why I say that the steering council mechanism will compromise the values of the problem-solving repo if it's given the power to rule over it.


> From this point of view I'd rather prefer dice-thrown approach. It's way more fair.

@vrurg Yeah, well, I know it may sound surprising, but this is not about being fair. Read up on [affirmative action](https://en.wikipedia.org/wiki/Affirmative_action). It is inherently unfair on purpose, with a hope that a reverse wrong will result in a better future. It's an interesting idea, and depending on where and how you apply it, it can be very useful (though in some countries it is outright illegal, for obvious reasons). But it is hard for me to argue whether it is applicable here, because I'm already being framed as a misogynist, even though I want to give *everyone* the right to stop proposals that would offend their minority. Life is weird.